workflows/__init__.py

```diff
@@ -2,6 +2,9 @@
 
 __version__ = "1.6"
 
+# Import validate function
+from workflows.validate import validate_recipe
```

Comment:
why?

Reply:
I thought this was a common thing so that someone using the library can do import workflows.validate_recipe rather than from workflows.validate import validate_recipe ?
Perhaps I just read it somewhere and misinterpreted the use?

Reply:
hmm you can do it, but given that the main use will be to invoke it via an entry point I don't think it is very useful here. Thinking about that I'm wondering - shouldn't `validate.py` really go into the `recipe/` directory?

Reply:
importing it into `recipe/__init__.py` would be less contentious. There is precent there.

Reply:
Yes, might be better to move both to recipe anyway? Seeing as it basically is a wrapper for the inner function of recipe.py

Reply:
Moved both to recipe module and made corresponding changes.